app/models/role_permission.rb

```diff
@@ -0,0 +1,4 @@
+class RolePermission < ActiveRecord::Base
```

Comment:
Shouldn't this be the `PermissionsRoles` class?

Reply:
Yes, It can be. I found this name more reliable. But we can change it. 👍

Reply:
Doesn't the Rails magic happen if it is `PermissionsRoles` ?